bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptDependencyTracker.java

```diff
@@ -79,9 +79,9 @@ public Path getLibraryPath() {
 
     @Override
     public void processWatchEvent(WatchService.Kind kind, Path path) {
-        File file = path.toFile();
+        File file = libraryPath.resolve(path).toFile();
         if (!file.isHidden() && (kind == DELETE || (file.canRead() && (kind == CREATE || kind == MODIFY)))) {
-            dependencyChanged(file.getPath());
+            dependencyChanged(path.toString());
```

Comment:
```suggestion
            dependencyChanged(file.toString());
```

I think `dependencyChanged` requires the absolute path and not the relative path to work, because `scriptToLibs` needs absolute paths:

![image](https://user-images.githubusercontent.com/73423173/228928983-f9c4879d-4077-4d47-b048-5db52d26a1be.png)

With the above change, the dependency tracker works again.

Reply:
Done